pkg/storage/db.go

```diff
@@ -21,6 +21,9 @@ const tableAWSEventsIPSHostnames = "aws_events_ips_hostnames"
 
 const added = "ADDED" // one of the network event types we track
 
+// test seam
+var sqlOpenFn = sql.Open
```

Comment:
Same question as with the component test since global state is one of the things we typically don't allow and it limits the ways we can run the tests.

Reply:
See comment above..